erpnext/buying/doctype/request_for_quotation/request_for_quotation.js

```diff
@@ -87,14 +87,20 @@ frappe.ui.form.on("Request for Quotation",{
 
 	},
 
-	make_suppplier_quotation: function(frm) {
+	make_supplier_quotation: function(frm) {
 		var doc = frm.doc;
 		var dialog = new frappe.ui.Dialog({
 			title: __("Create Supplier Quotation"),
 			fields: [
 				{	"fieldtype": "Select", "label": __("Supplier"),
 					"fieldname": "supplier",
-					"options": doc.suppliers.map(d => d.supplier),
+					"options": doc.suppliers.map(d => {
```

Comment:
@ahmadpak just make this field a link field. Users will be able to configure what they want to see as search fields and supplier names will also be by default visible

Reply:
@deepeshgarg007 Thank you for the feedback. will do so